set_window_title.py

```diff
@@ -143,7 +144,9 @@ def rename_window_linux(self, window, official_title, new_title, debug=False):
   def rename_window_windows(self, new_title):
     # PX_WINDOW_CLASS is the ClassName of SublimeText, can be seen via a tool such as Nirsoft WinLister
     hwndSublime = ctypes.windll.user32.FindWindowA(b'PX_WINDOW_CLASS', None)
-    ctypes.windll.user32.SetWindowTextW(hwndSublime, new_title)
+    text = new_title.encode("utf16")	
+    buff = (ctypes.c_char * (len(text)+10))(*text)
```

Comment:
Thanks for the PR, I'm hoping this helps me, too.

Where does this 10 come from?

Reply:
I took it from the title setter method that was removed in the last commit. 
https://github.com/gwenzek/SublimeSetWindowTitle/commit/14e4d0bae8704d7b59e7856bba341de9cc86a94c#diff-a7908ba95124deb7948b618398904c6eL66

I imagine it was just there for a bit of breathing room.